vocab-builder-maven-plugin/src/main/java/com/github/tkurz/sesame/vocab/plugin/VocabularyBuilderMojo.java

```diff
@@ -314,6 +413,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             throw new MojoExecutionException("Could not write Vocabularies", e);
         }
     }
+    /**
+     * Helper method for initializing based on the provided caseFormat string 
+     * @param caseFormat
+     * @return
+     */
+    private CaseFormat caseFormatFromString(String caseFormat) {
```

Comment:
This code should not be replicated both here and in GenerationSetting. Need to have a static method in a helper class so that we ensure that we are doing the same operations everywhere for this case.